.github/workflows/test.yml

```diff
@@ -182,7 +182,7 @@ jobs:
     needs: e2e-ios-build
     timeout-minutes: 15
     env:
-      DEVELOPER_DIR: /Applications/Xcode_11.4.app
+      DEVELOPER_DIR: /Applications/Xcode_11.5.app
```

Comment:
noticing this one is finding the application via setting the env.  We could probably change it to use the xcode-select utility, but I'm kinda hesitant to make more changes to build pipeline stuff than is necessary.

Reply:
I'd say if it passes with the same command as the other tools that's fine by me :) It may be that detox requires the environment variables set, but the easiest way to confirm that would be a test run, I think :)